### PR TITLE
Bugfix/update dead links

### PR DESCRIPTION
**Pull request by jbern0rd**
- [ ] My code has been formatted as explained in [CONTRIBUTING.md](https://github.com/iExecBlockchainComputing/documentation/blob/v8-staging/CONTRIBUTING.md)

### CODE REVIEW

File: for-developers/quick-start-for-developers.md [hunk 1]
```diff
@@ -88,7 +88,8 @@ You can now connect to the blockchain. In the following steps we will use the **
 
 {% hint style="info" %}
 
-**Bellecour** is an Ethereum blockchain run by iExec. Unlike the Ethereum mainnet or public testnets, transactions on Bellecour do not require the payment of transaction fees ([Read more about transaction fees](https://bitfalls.com/2017/12/05/ethereum-gas-and-transaction-fees-explained/)).
+**Bellecour** is an Ethereum blockchain run by iExec. Unlike the Ethereum mainnet or public testnets,
```
Comment (ericr6) [inline for-developers/quick-start-for-developers.md]: Suggestion:
[Read more about iExec Sidechain] (https://docs.iex.ec/for-developers/iexec-sidechain/overview)

File: for-developers/quick-start-for-developers.md [hunk 2]
```diff
@@ -88,7 +88,8 @@ You can now connect to the blockchain. In the following steps we will use the **
...
+transactions on Bellecour do not require the payment of transaction fees ([Read more about transaction fees](https://ethereum.org/en/developers/docs/gas/)).
```
Comment (ericr6) [inline for-developers/quick-start-for-developers.md]: move to (https://docs.iex.ec/for-developers/iexec-sidechain/overview) ?

File: for-developers/quick-start-for-developers.md (hunk 1 above)
Comment (jbern0rd) [reply for-developers/quick-start-for-developers.md]: I tried something, please see bb02442

File: for-developers/quick-start-for-developers.md (hunk 2 above)
Comment (jbern0rd) [reply for-developers/quick-start-for-developers.md]: After discussion with sidechain team, the link was not moved to sidechain overview.
People should be able to look by themselves after this kind of ressources.

**Review by ericr6 — APPROVED** [review]
good

**Review by Mikexec — APPROVED** [review]

**Pull request merged**